src/test/java/Homework18.java

```diff
@@ -19,4 +19,4 @@ public void playSong()throws InterruptedException {
         clickPlay();
         Assert.assertTrue(isSongPlaying());
     }
-}
\ No newline at end of file
+}
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![Screenshot_2](https://user-images.githubusercontent.com/104761482/225237061-88fb45fd-0065-4247-9434-570e75783df9.jpg)